src/shared/lib/fetch/get.js

```diff
@@ -65,13 +69,20 @@ export const get = async (url, type, date = datetime.old.scrapeDate() || datetim
 
       // TODO @AWS: if AWS infra get from endpoint instead of needle
 
+      // Errors we get here have the tendency of crashing the whole crawler
+      // with no ability for us to catch them. Let's hear what these errors have to say,
+      // and throw an error later down that won't bring the whole process down.
       let errorMsg = '';
-      const response = await needle('get', url, { cookies, headers }).catch(err => {
-        // Errors we get here have the tendency of crashing the whole crawler
-        // with no ability for us to catch them. Let's hear what these errors have to say,
-        // and throw an error later down that won't bring the whole process down.
-        errorMsg = err.toString();
-      });
+      let response;
+      if (method === 'get') {
+        response = await needle('get', url, { cookies, headers }).catch(err => {
+          errorMsg = err.toString();
+        });
+      } else {
+        response = await needle('post', url, args, { cookies, headers }).catch(err => {
```

Comment:
```suggestion
        response = await needle(method, url, args, { cookies, headers }).catch(err => {
```

Reply:
Kinda seems like we could just pass `method` at all times

Reply:
But `needle('get', ...)` doesn't take args. Only `needle('post', ...)` does.